silkrpc/http/request_handler.cpp

```diff
@@ -24,86 +24,104 @@
+    handlers_[method::k_eth_getTransactionByBlockNumberAndIndex] = &commands::RpcApi::handle_eth_get_transaction_by_block_number_and_index;
+    handlers_[method::k_eth_getTransactionReceipt] = &commands::RpcApi::handle_eth_get_transaction_receipt;
+    handlers_[method::k_eth_estimateGas] = &commands::RpcApi::handle_eth_estimate_gas;
+    handlers_[method::k_eth_getBalance] = &commands::RpcApi::handle_eth_get_balance;
+    handlers_[method::k_eth_getCode] = &commands::RpcApi::handle_eth_get_code;
+    handlers_[method::k_eth_getTransactionCount] = &commands::RpcApi::handle_eth_get_transaction_count;
+    handlers_[method::k_eth_getStorageAt] = &commands::RpcApi::handle_eth_get_storage_at;
+    handlers_[method::k_eth_call] = &commands::RpcApi::handle_eth_call;
+    handlers_[method::k_eth_newFilter] = &commands::RpcApi::handle_eth_new_filter;
+    handlers_[method::k_eth_newBlockFilter] = &commands::RpcApi::handle_eth_new_block_filter;
+    handlers_[method::k_eth_newPendingTransactionFilter] = &commands::RpcApi::handle_eth_new_pending_transaction_filter;
+    handlers_[method::k_eth_getFilterChanges] = &commands::RpcApi::handle_eth_get_filter_changes;
+    handlers_[method::k_eth_uninstallFilter] = &commands::RpcApi::handle_eth_uninstall_filter;
+    handlers_[method::k_eth_getLogs] = &commands::RpcApi::handle_eth_get_logs;
+    handlers_[method::k_eth_sendRawTransaction] = &commands::RpcApi::handle_eth_send_raw_transaction;
+    handlers_[method::k_eth_sendTransaction] = &commands::RpcApi::handle_eth_send_transaction;
+    handlers_[method::k_eth_signTransaction] = &commands::RpcApi::handle_eth_sign_transaction;
+    handlers_[method::k_eth_getProof] = &commands::RpcApi::handle_eth_get_proof;
+    handlers_[method::k_eth_mining] = &commands::RpcApi::handle_eth_mining;
+    handlers_[method::k_eth_coinbase] = &commands::RpcApi::handle_eth_coinbase;
+    handlers_[method::k_eth_hashrate] = &commands::RpcApi::handle_eth_hashrate;
+    handlers_[method::k_eth_submitHashrate] = &commands::RpcApi::handle_eth_submit_hashrate;
+    handlers_[method::k_eth_getWork] = &commands::RpcApi::handle_eth_get_work;
+    handlers_[method::k_eth_submitWork] = &commands::RpcApi::handle_eth_submit_work;
+    handlers_[method::k_eth_subscribe] = &commands::RpcApi::handle_eth_subscribe;
+    handlers_[method::k_eth_unsubscribe] = &commands::RpcApi::handle_eth_unsubscribe;
+    handlers_[method::k_eth_getBlockReceipts] = &commands::RpcApi::handle_parity_get_block_receipts;
+}
+
+void RequestHandler::add_net_handlers() {
+    handlers_[method::k_net_listening] = &commands::RpcApi::handle_net_listening;
+    handlers_[method::k_net_peerCount] = &commands::RpcApi::handle_net_peer_count;
+    handlers_[method::k_net_version] = &commands::RpcApi::handle_net_version;
+}
+
+void RequestHandler::add_parity_handlers() {
+    handlers_[method::k_parity_getBlockReceipts] = &commands::RpcApi::handle_parity_get_block_receipts;
+}
+
+void RequestHandler::add_tg_handlers() {
```

Comment:
should it not be `erigon_`? just curiosity, not relevant to the PR's scope

Reply:
Yes, there's already an issue for that: see #101